website/static/data/plugin.json

```diff
@@ -145,6 +145,10 @@
             {
                 "name": "zipkin",
                 "description": "Zipkin is a OpenTracing plugin"
+            },
+            {
+                "name": "datadog",
+                "description": "Datadog is a monitoring plugin built into Apache APISIX for seamless integration with Datadog"
```

Comment:
Hi @bisakhmondal @yzeng25, do we have a better description for it?

Reply:
How about 
> `Datadog` is a monitoring plugin built into Apache APISIX to push metrics of different services into DatadogHQ via a locally run Datadog agent.

Thanks!